src/Pos/Generator/Block/Mode.hs

```diff
@@ -123,7 +124,7 @@ data BlockGenContext = BlockGenContext
     , bgcSystemStart       :: !Timestamp
     , bgcParams            :: !BlockGenParams
     , bgcDelegation        :: !DelegationVar
-    , bgcGenesisUtxo       :: !GenesisUtxo
+    , bgcGenesisTxpContext :: !GenesisTxpContext
```

Comment:
didn't notice that it's block-gen, ok, I can remove. but it can be needed in the future